Makefile

```diff
@@ -500,7 +500,7 @@ build_saturn_link_native: build_saturn_link_copy build_saturn_link_native_ld
 build_saturn_docker: build_saturn_copy_files build_saturn_dosemu_docker build_saturn_link_docker
 
 # do not run in parallel
-build_saturn_native: build_saturn_copy_files build_saturn_dosemu_native build_saturn_link_native
+build_saturn_native: build_saturn_copy_files build_saturn_dosemu_native build_saturn_link_native check_saturn_native
```

Comment:
What do you mean by it isn't consistent? it is exactly how the PSX build works, it shows the checksum of every file after building.

Reply:
@sonicdcer What command are you running to get a build and check? `make all`? If you do `make build` on the PSX port you don't get the check behavior. I approved the Makefile change originally because I don't really care that much either way. However, as far as I understand the current behavior on master is closer to how the build system for PSX works.

Reply:
I'm using build_saturn_native, which for me is the current equivalent of make all for PSX, i know we need to change the names so i included the check inside the build_saturn_native for now so it behaves correctly for the time being until we decide on name commands. I would appreciate if @Xeeynamo explained why he thinks this is incorrect or proposing an alternative instead of just demanding a revert as a condition to merge the PR.

Reply:
Ok so what you are asking for is an equivalent of `make all` on the Saturn version? I usually do `make build` on the PSX port so I don't get the check behavior. I wrote `make build_saturn_native` to be equivalent to `make build` on the PSX version.

Reply:
Yeah, that's what we want, a command to build and check.